installers/common.sh

```diff
@@ -207,6 +207,7 @@ function _setup_synchrona_service() {
     ExecStartPre=-/usr/bin/dtoverlay /boot/overlays/rpi-ad9545-hmc7044.dtbo
     ExecStartPre=-/usr/bin/sh /var/www/html/app/python/synchrona/rebind.sh
     ExecStart=/usr/local/bin/uvicorn --app-dir=/var/www/html/app/python/synchrona  main:app --host 0.0.0.0 --port 8000
+    ExecStartPost=-/usr/bin/bash /var/www/html/app/python/synchrona/ad-synchrona-14-leds.sh > /dev/null 2>&1 &
```

Comment:
nope... we should not be doing this like this. You should not hardcode `/var/www/html/` as the user might choose a different installation path. I do have a fix for this in [here](https://github.com/analogdevicesinc/raspap-webgui/pull/2/commits/1472a9c51273dd67b2ca8536a7f42fada704bf88). It will be merged in the next few days...

I would actually move your script to installers/ and during installation move it to /etc/raspap/synchrona since, AFAICT, that path is not configurable by the user.